hack/jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -0,0 +1,168 @@
+        - string:
+            name: sha1
+            description: 'sha1 or refname (e.g. origin/pr/N/head) to build'
+    scm:
+        - git:
+            remotes:
+                - remote:
+                    url: https://github.com/kubernetes/kubernetes
+                    refspec: '+refs/heads/*:refs/remotes/upstream/*'
+                - remote:
+                    url: https://github.com/kubernetes/kubernetes
+                    refspec: '+refs/pull/${ghprbPullId}/merge:refs/remotes/origin/pr/${ghprbPullId}/merge'
+            branches:
+                - 'origin/pr/${ghprbPullId}/merge'
+            browser: githubweb
+            browser-url: http://github.com/kubernetes/kubernetes
+            timeout: 20
+            clean:
+                after: true
+            wipe-workspace: false
+            skip-tag: true
+    triggers:
+        - github-pull-request:
+            # This is the Jenkins GitHub ID, not the actual github token.
+            auth-id: 'f8e31bc1-9abb-460a-a2ca-9c4aae3ca4e8'
+            trigger-phrase: '(?is).*@k8s-bot\s+unit\s+test\s+this.*'
+            cron: 'H/2 * * * *'
+            status-context: Jenkins unit/integration
+            status-url: 'http://pr-test.k8s.io/${ghprbPullId}/${JOB_NAME}/${BUILD_NUMBER}/'
+            admin-list:
+                - alex-mohr
+                - amygdala
+                - a-robinson
+                - ArtfulCoder
+                - bgrant0607
+                - bgrant0607-nocc
+                - bprashanth
+                - brendandburns
+                - caesarxuchao
+                - cjcullen
+                - davidopp
+                - dchen1107
+                - ddysher
+                - derekwaynecarr
+                - eparis
+                - erictune
+                - fabioy
+                - fgrzadkowski
+                - gmarek
+                - goltermann
+                - ihmccreery
+                - ixdy
+                - JanetKuo
+                - jbeda
+                - jdef
+                - jlowdermilk
+                - jszczepkowski
+                - justinsb
+                - k8s-merge-robot
+                - karlkfi
+                - kelseyhightower
+                - krousey
+                - lavalamp
+                - liggitt
+                - mansoorj
+                - markturansky
+                - mbforbes
+                - mikedanese
+                - MikeJeffrey
+                - mwielgus
+                - nikhiljindal
+                - piosz
+                - pmorie
+                - pwittrock
+                - quinton-hoole
+                - RichieEscarez
+                - rjnagal
+                - roberthbailey
+                - saad-ali
+                - satnam6502
+                - smarterclayton
+                - thockin
+                - vishh
+                - vmarmol
+                - wojtek-t
+                - yujuhong
+                - zmerlynn
+            white-list:
```

Comment:
ok, I've updated the PR Jenkins config/comment to note this. (it will be a little confusing for a bit, since this config isn't checked in, but I'll field the complaints.)